### PR TITLE
Update retry delay time and calculation

### PR DESCRIPTION
**Pull request by 0xKrishna**

### CODE REVIEW

File: bridge/setu/processor/staking.go [hunk 1]
```diff
@@ -355,7 +355,11 @@ func (sp *StakingProcessor) checkValidNonce(validatorId uint64, txnNonce uint64)
 
 	if currentNonce+1 != txnNonce {
 		sp.Logger.Error("Nonce for the given event not in order", "validatorId", validatorId, "currentNonce", currentNonce, "txnNonce", txnNonce)
-		return false, txnNonce - currentNonce, nil
+		diff := txnNonce - currentNonce
```
Comment (JekaMas) [inline bridge/setu/processor/staking.go:358]: Could we move logging a bit lower and also log the delay value?

Comment (0xKrishna) [reply bridge/setu/processor/staking.go:358]: Sure

File: bridge/setu/processor/staking.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	defaultDelayDuration time.Duration = 10 * time.Second
+	defaultDelayDuration time.Duration = 15 * time.Second
```
Comment (JekaMas) [inline bridge/setu/processor/staking.go:21]: Could we create an issue to move such hard coded values to a config and pass the config either as a parameter or as a value in context?

Comment (0xKrishna) [reply bridge/setu/processor/staking.go:21]: Sure, Will create an issue for this.

File: bridge/setu/processor/staking.go (hunk 1 above)
Comment (0xKrishna) [reply bridge/setu/processor/staking.go:357]: Done

File: bridge/setu/processor/staking.go [hunk 3]
```diff
@@ -355,7 +355,11 @@ func (sp *StakingProcessor) checkValidNonce(validatorId uint64, txnNonce uint64)
 
 	if currentNonce+1 != txnNonce {
 		sp.Logger.Error("Nonce for the given event not in order", "validatorId", validatorId, "currentNonce", currentNonce, "txnNonce", txnNonce)
-		return false, txnNonce - currentNonce, nil
+		diff := txnNonce - currentNonce
+		if diff > 10 {
+			return false, 10, nil
```
[the remark this answers is not in the document]
Comment (0xKrishna) [reply bridge/setu/processor/staking.go:359]: Done

**Review by JekaMas — APPROVED** [review]

**Review by manav2401 — APPROVED** [review]

**Pull request merged**